v19.2/window-functions.md

```diff
@@ -38,112 +91,31 @@ Its operation can be described as follows (numbered steps listed here correspond
 
 1. The outer `SELECT DISTINCT(city) ... FROM rides` creates a "virtual table" on which the window functions will operate.
 2. The window function `SUM(revenue) OVER ()` operates on a window frame containing all rows of the query output.
-3. The window function `SUM(revenue) OVER (PARTITION BY city)` operates on several window frames in turn; each frame contains the `revenue` columns for a different city (Amsterdam, Boston, L.A., etc.).
+3. The window function `SUM(revenue) OVER (PARTITION BY city)` operates on several window frames in turn; each frame contains the `revenue` columns for a different city [partition](partitioning.html) (Amsterdam, Boston, L.A., etc.).
```

Comment:
I think my comment above re: this is not that `PARTITION BY` also applies here (but we can wait for Yahor to confirm).  It's unfortunate that movr is fundamentally geo-distributed and this particular syntax is overloaded since it makes it quite confusing IMO.